scraper-tools/src/browser_tools.js

```diff
@@ -165,17 +165,19 @@ const saveSnapshot = async ({ page, $ }) => {
     }
     lastSnapshotTimestamp = now;
 
-    if ($) {
-        await Apify.setValue(SNAPSHOT.KEYS.HTML, $.html(), { contentType: 'text/html' });
-    }
-    if (page) {
+    if (body && contentType) {
+        const rawBody = (typeof body === 'object') ? JSON.stringify(body) : body;
```

Comment:
This got me thinking. So, originally, we had `$` with parsed HTML and `html` with raw HTML string. Now we have `body` which will contain either **raw** HTML string or **parsed** JSON or a `Buffer`?

Reply:
Feels really weird. Perhaps have always a raw `body` and in case of JSON, add a `json` getter that just does `JSON.parse()` so that the user doesn't have to do it themselves?